modules/conversion-center/assets/js/editor/hooks/ui/editor/save/set-is-modified.js

```diff
@@ -0,0 +1,22 @@
+import HookUIAfter from 'elementor-api/modules/hooks/ui/after';
+
+export class FooterSaverActiveCopyAndShare extends HookUIAfter {
+	getCommand() {
+		return 'document/save/set-is-modified';
+	}
+
+	getId() {
+		return 'footer-saver-activate-copy-share';
+	}
+
+	apply( args ) {
```

Comment:
Do you need to add conditions here?

Reply:
@louiswol94 same as above, refactoring it.

Reply:
@louiswol94 added them